assets/js/base/context/cart-checkout/checkout-state/reducer.js

```diff
@@ -66,9 +67,9 @@ export const prepareResponseData = ( data ) => {
  */
 export const reducer = (
 	state = DEFAULT_STATE,
-	{ url, type, orderId, data }
+	{ url, type, orderId, orderNotes, data }
 ) => {
-	let newState;
+	let newState = state;
```

Comment:
With the changes in 66f3d3330ca019baaf3f8a37e3d1fe0ce6498a4f, for example, it's not always sure we will assign a value to `newState`, so it's good to initialize it to the previous state to avoid unexpected errors.